### PR TITLE
Fix 597

### PR DESCRIPTION
**Pull request by Tropicalista**

### CODE REVIEW

**Comment by lmajano** [comment]
Which ticket is this for @Tropicalista

**Comment by lmajano** [comment]
Forget it found it

File: system/core/dynamic/HTMLHelper.cfc [hunk 1]
```diff
@@ -86,7 +86,7 @@ Description :
 				if( NOT listFindNoCase(event.getValue(name="cbox_assets",private=true),thisAsset) ){
 
 					// Load Asset
-					if( findNoCase(".js", thisAsset) ){
+					if( findNoCase( "js", listLast(thisAsset,".") ) ){
```
Comment (bdw429s) [inline system/core/dynamic/HTMLHelper.cfc]: This won't work if a value in the query string has a period in it.
```site.com/assets/myFile.js?foo=bar.baz```
See my comments in the Box team Slack channel.
cc/ @lmajano

**Comment by lmajano** [comment]
So I see two pull requests with the same ticket? Which one is the one to use?

**Pull request closed without merging**